_source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md

```diff
@@ -10,127 +10,78 @@ communities: [javascript]
 description: "In this article you'll learn how to use OpenID Connect to build a simple Node.js website with Express.js. You'll also learn how to log users into your website using Okta."
 tags: [javascript, oidc, openid, nodejs, expressjs]
 tweets:
-    - "Learn how to use OpenID Connect to build a simple Node website."
+    - "Learn how to use OpenID Connect to build a simple Node website 🛠"
     - "Come see how to build a simple Node website using OpenID Connect!"
 type: conversion
+image: blog/use-openid-connect-to-build-a-simple-node-website/openid-node-social.jpg
+github: https://github.com/oktadev/okta-express-example
+changelog:
+  - 2022-01-20: Updated dependencies, code, and the cover image. See this post's changes in [okta-blog#1043](https://github.com/oktadev/okta-blog/pull/1043) and the example app changes in [okta-express-example#3](https://github.com/oktadev/okta-express-example/pull/3).
 ---
 
 
 If you've ever spent time trying to figure out the best way to handle user
 authentication for your Node app and been confused: you're not alone. Over the
-last few years authentication practices have changed quite a bit.
+last few years, authentication practices have changed quite a bit.
 
 Today I'm going to show you how to use OpenID Connect to build an extremely
 simple Node.js website (using Express.js) that allows you to manage your users,
 log them in, and log them out.
 
-Back in the day, all websites would require users to register with a
-username/password and log in with those same credentials. This was simple, but
-caused a lot of security problems because developers would need to write the
+All websites would require users to register with a
+username/password and log in with those same credentials back in the day. This method was simple but
```

Comment:
```suggestion
username/password and log in with those same credentials. This method was simple but
```
"back in the day" seems to dangle as a closing phrase, but I don't think it's necessary here.